cli/command/context_test.go

```diff
@@ -0,0 +1,42 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/docker/cli/cli/context/store"
+	"gotest.tools/assert"
+)
+
+func TestContextStorageParsingAndSaving(t *testing.T) {
+	cases := []struct {
+		name string
+		data ContextMetadata
+	}{
+		{
+			name: "empty",
+			data: ContextMetadata{},
+		},
+		{
+			name: "with-values",
+			data: ContextMetadata{
+				Description:       "description",
+				StackOrchestrator: OrchestratorKubernetes,
+			},
+		},
+	}
+
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			storeMeta := store.ContextMetadata{}
+			SetContextMetadata(&storeMeta, c.data)
+			result, err := GetContextMetadata(storeMeta)
+			assert.Check(t, err)
```

Comment:
I don't understand this check. Use `assert.NilError` or `assert.Error` instead?

Reply:
seem to be equivalent to assert.NilError. but will do.